README.md

```diff
@@ -2,42 +2,98 @@ PReview Code frontend [![Build Status](https://travis-ci.org/preview-code/fronte
 =========
 This is the frontend code for https://preview-code.com implemented with Polymer.
 
-# Installation for testing
+Please read the complete README.md file before executing any of the steps.
 
-Make sure you have `web-component-tester` installed globally as well as
-the plugin `web-component-tester-istanbul`:
+If you live in the Netherlands, please consider using the following mirror in "Ubuntu Software & Updates":
```

Comment:
The scripts require Bash, not Ubuntu. You can run them just fine on Mac, so you can remove this requirement.